exporter/otlphttpexporter/config_test.go

```diff
@@ -67,10 +69,14 @@ func TestUnmarshalConfig(t *testing.T) {
 					},
 					Insecure: true,
 				},
-				ReadBufferSize:  123,
-				WriteBufferSize: 345,
-				Timeout:         time.Second * 10,
-				Compression:     "gzip",
+				ReadBufferSize:      123,
+				WriteBufferSize:     345,
+				Timeout:             time.Second * 10,
+				Compression:         "gzip",
+				MaxIdleConns:        &defaultTransport.MaxIdleConns,
+				MaxIdleConnsPerHost: &defaultTransport.MaxIdleConnsPerHost,
+				MaxConnsPerHost:     &defaultTransport.MaxConnsPerHost,
+				IdleConnTimeout:     &defaultTransport.IdleConnTimeout,
```

Comment:
This is not ok, because you may change the defaults now correct? Cannot use pointer to that, we should have `var defaultIdleConnTimeout = defaultTransport.IdleConnTimeout`.

Reply:
youre right, updated in last commit 👍